src/main/java/util/Ui.java

```diff
@@ -0,0 +1,101 @@
+package util;
+
+import java.util.Scanner;
+
+import action.Action;
+import action.Add;
+import action.Delete;
+import action.Find;
+import action.Goodbye;
+import action.HandleError;
+import action.List;
+import action.Mark;
+import action.Unmark;
+import exception.NarutoException;
+import task.Deadline;
+import task.Event;
+import task.ToDo;
+
+/**
+ * The Ui class handles user interface related operations.
+ */
+public class Ui {
+    private static final Scanner sc = new Scanner(System.in);
+
+    private Ui() {
+
+    }
+
+    /**
```

Comment:
It is good that you ensure all your public classes and methods have Javadoc comments, and they follow the conventions of the Java coding standard, nice!